homeassistant/components/meater/sensor.py

```diff
@@ -99,7 +103,9 @@ def _remaining_time_to_timestamp(probe: MeaterProbe) -> datetime | None:
         native_unit_of_measurement=TEMP_CELSIUS,
         state_class=SensorStateClass.MEASUREMENT,
         available=lambda probe: probe is not None and probe.cook is not None,
-        value=lambda probe: probe.cook.target_temperature if probe.cook else None,
+        value=lambda probe: probe.cook.target_temperature
```

Comment:
It's easier to read a standalone function if the lambda spans more than one line.